packages/grpc-js/package.json

```diff
@@ -57,7 +60,7 @@
   "dependencies": {
     "@grpc/proto-loader": "^0.6.0-pre14",
     "@types/node": "^12.12.47",
-    "google-auth-library": "^6.0.0",
+    "google-auth-library": "^5.10.1",
```

Comment:
Are we downgrading this? Wasn't it bumped recently?

Reply:
Having it at 6 was an error, because 6 requires Node 10 and above (#1442). I mistakenly bumped it up to 6 while moving it around between prod dependencies and peer dependencies and dev dependencies.